{
  "repository": {
    "name": "hekate-io/hekate",
    "githubId": 84603051,
    "url": "https://github.com/hekate-io/hekate",
    "isFork": false,
    "original": {
      "name": "",
      "githubId": 0,
      "url": ""
    },
    "isPullRequest": false,
    "pullRequestId": 0
  },
  "builds": {
    "buggyBuild": {
      "id": 412177998,
      "url": "http://travis-ci.org/hekate-io/hekate/builds/412177998",
      "date": "Aug 5, 2018 12:12:40 AM"
    }
  },
  "projectMetrics": {
    "numberModules": 13,
    "numberSourceFiles": 652,
    "numberTestFiles": 351,
    "numberLibrariesFailingModule": 29
  },
  "tests": {
    "failingModule": "/root/workspace/hekate-io/hekate/412177998/hekate-core",
    "overallMetrics": {
      "numberRunning": 9385,
      "numberPassing": 9332,
      "numberFailing": 1,
      "numberErroring": 52,
      "numberSkipping": 64,
      "failures": [
        {
          "failureName": "org.postgresql.util.PSQLException",
          "isError": true,
          "occurrences": 26
        },
        {
          "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
          "isError": true,
          "occurrences": 26
        },
        {
          "failureName": "java.lang.AssertionError",
          "isError": false,
          "occurrences": 1
        }
      ]
    },
    "failingClasses": [
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "numberRunning": 39,
        "numberPassing": 13,
        "numberFailing": 0,
        "numberErroring": 26,
        "numberSkipping": 0
      },
      {
        "testClass": "io.hekate.cluster.split.JdbcConnectivityDetectorTest",
        "numberRunning": 6,
        "numberPassing": 5,
        "numberFailing": 1,
        "numberErroring": 0,
        "numberSkipping": 0
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "numberRunning": 39,
        "numberPassing": 13,
        "numberFailing": 0,
        "numberErroring": 26,
        "numberSkipping": 0
      }
    ],
    "failureDetails": [
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testGetCleanupInterval[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testUnregisterUnknown[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testStartStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testMultipleStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStartDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testCleanupWithNodes[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testStartStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testCleanupWithNodes[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterDuplicate[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregister[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnGetNodes[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testUnregisterUnknown[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterDuplicate[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testGetCleanupInterval[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterDuplicate[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testMultipleStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStartDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testUnregisterUnknown[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testToString[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testGetCleanupInterval[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.split.JdbcConnectivityDetectorTest",
        "testMethod": "testRealDataSources",
        "failureName": "java.lang.AssertionError",
        "detail": "com.mysql.cj.jdbc.MysqlDataSource@c66980",
        "isError": false
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregister[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregisterWrongCluster[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDiscovery[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDiscovery[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregister[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterDuplicate[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testUnregisterUnknown[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testToString[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnGetNodes[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testCleanupWithNodes[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testMultipleStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnGetNodes[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testCleanupWithNodes[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregister[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testStartStopDiscovery[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnGetNodes[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testMultipleStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregisterWrongCluster[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testToString[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testToString[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testRegisterUnregisterWrongCluster[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testDbErrorOnStartDiscovery[2:org.postgresql.ds.PGSimpleDataSource@68226b59]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testRegisterUnregisterWrongCluster[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testGetCleanupInterval[1:com.mysql.cj.jdbc.MysqlDataSource@4823f3b8]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderCustomTableTest",
        "testMethod": "testDbErrorOnStartDiscovery[2:org.postgresql.ds.PGSimpleDataSource@85446a]",
        "failureName": "org.postgresql.util.PSQLException",
        "detail": "Connection to 127.0.01:5433 refused. Check that the hostname and port are correct and that the postmaster is accepting TCP/IP connections.",
        "isError": true
      },
      {
        "testClass": "io.hekate.cluster.seed.jdbc.JdbcSeedNodeProviderTest",
        "testMethod": "testStartStopDiscovery[1:com.mysql.cj.jdbc.MysqlDataSource@cb29b75]",
        "failureName": "com.mysql.cj.jdbc.exceptions.CommunicationsException",
        "detail": "Communications link failure\n\nThe last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.",
        "isError": true
      }
    ]
  },
  "reproductionBuggyBuild": {
    "reproductionDateBeginning": "Aug 5, 2018 12:13:27 AM",
    "reproductionDateEnd": "Aug 5, 2018 1:26:27 AM",
    "totalDuration": 4376,
    "processDurations": {
      "cloning": {
        "nbSteps": 1,
        "totalDuration": 5,
        "stepNames": [
          "CloneRepository"
        ],
        "stepDurations": [
          5
        ]
      },
      "building": {
        "nbSteps": 2,
        "totalDuration": 189,
        "stepNames": [
          "CheckoutBuggyBuild",
          "BuildProject"
        ],
        "stepDurations": [
          0,
          189
        ]
      },
      "testing": {
        "nbSteps": 1,
        "totalDuration": 2372,
        "stepNames": [
          "TestProject"
        ],
        "stepDurations": [
          2372
        ]
      }
    },
    "projectRootPomPath": "./workspace/hekate-io/hekate/412177998/pom.xml"
  },
  "patchDiff": {
    "files": {
      "numberAdded": 0,
      "numberChanged": 0,
      "numberDeleted": 0
    },
    "lines": {
      "numberAdded": 0,
      "numberDeleted": 0
    }
  },
  "commits": {
    "buggyBuild": {
      "repoName": "hekate-io/hekate",
      "branchName": "development",
      "sha": "e95c468f7e21a77a79dbcfd81b2f7483f3ac7ff0",
      "url": "http://github.com/hekate-io/hekate/commit/e95c468f7e21a77a79dbcfd81b2f7483f3ac7ff0",
      "date": "Aug 4, 2018 11:24:16 PM"
    }
  },
  "type": "only_fail"
}